config/main_window.json

```
@@ -0,0 +1,19 @@
+{
+    "window_xml_file": "./gtk_gui/xml/welcome_window.glade",
+    "window_size_x": "700",
+    "window_size_y": "500",
+    "window_fullscreen": true,
+    "window_logo_bot_path": "./img/raven_title_white.png",
+    "window_list_components": [
+        "welcome_window",
+        "welcome_window_prev_btn",
+        "welcome_window_next_btn",
+        "welcome_window_left_top_logo",
+        "welcome_window_left_bot_logo",
+        "welcome_window_left_bot_label",
+        "welcome_window_right_header_label",
```

```suggestion
        "welcome_window_header_right_label",
```

Same here, it's usually Y axis then X axis when naming things (we say the top-right corner, not the right-top corner)